wire-ios-data-model/Source/MLS/OneOnOne/OneOnOneResolver.swift

```diff
@@ -43,30 +44,28 @@ public final class OneOnOneResolver: OneOnOneResolverInterface {
 
     private let protocolSelector: OneOnOneProtocolSelectorInterface
     private let migrator: OneOnOneMigratorInterface?
+    private let mlsService: MLSServiceInterface?
```

Comment:
Why is it optional? When does it make sense to initialise a OneOnOneResolve without a `MLSService`?

Reply:
You can init a OneOnOneResolver if you have only proteus and the other user only mls. The resolver then sets the 1-1 conversation to read only.

Reply:
Solved by other changes